src/main/java/org/jetbrains/bsp/bazel/server/ScalaBspServer.java

```diff
@@ -101,4 +121,18 @@ public ScalaBspServer(
                     inputs,
                     Uri.fromExecPath(Constants.EXEC_ROOT_PREFIX + output, execRoot).toString()));
   }
+
+  private List<ScalaMainClass> collectMainClasses(
+      BuildTargetIdentifier target,
+      Map<String, List<String>> mainClassesNames,
+      Map<String, List<String>> targetsOptions) {
+    return mainClassesNames.getOrDefault(target.getUri(), Lists.newArrayList()).stream()
+        .map(
+            mainClassName ->
+                new ScalaMainClass(
+                    mainClassName,
+                    new ArrayList<>(),
```

Comment:
nitpick: sometimes you use `Lists.newArrayList()` and sometimes `new ArrayList<>()`